src/main/scala/beam/sim/population/PopulationAdjustment.scala

```diff
@@ -229,6 +224,14 @@ object PopulationAdjustment extends LazyLogging {
           agentsim.agents.Population.getVehiclesFromHousehold(household, beamServices)
         HouseholdAttributes(household, houseHoldVehicles)
     }
+    // Read person attribute "valueOfTime", use function of HH income if not, and default it to the respective config value if neither is found
+    val valueOfTime: Double =
+      Option(personAttributes.getAttribute(person.getId.toString, "valueOfTime"))
+        .map(_.asInstanceOf[Double])
```

Comment:
This should probably just have been a `toDouble`, but maybe there is a reason to use `asInstanceOf`?